src/components/logs/logsWebview.ts

```diff
@@ -51,6 +51,8 @@ export class LogsPanel extends WebPanel {
                 </select>
                 <span style='position: absolute; left: 240px'>Match expression</span>
                 <input style='left:350px; position: absolute' type='text' id='regexp' onkeyup='eval()' placeholder='Filter' size='25'/>
+                <span style='position: absolute; left: 600px'>Auto-scroll</span>
+                <input style='left: 675px; position: absolute' type='checkbox' id='autoscroll' onkeyup='eval()' checked="true"/>
```

Comment:
This makes the checkbox appear even in Show Logs (non-tailing) mode.  I'm not sure this makes sense.  Could we hide the caption and check box in that mode?  (The way it's done at the moment, the panel doesn't know whether the user is showing or following, so we'd need the controlling command to pass that info in I guess.)

Reply:
We just have it showing up in all cases currently. There wasn't a clear way to identify when we were coming in via show or follow.